scripts/run_model_in_rocm_docker.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+MODEL="BERT-Large"
+#IMAGE="rocm/tensorflow:latest"
+IMAGE="rocm/tensorflow:rocm3.1-tf1.15-dev"
+
+# Print a message
+echo "This script will run the $MODEL model in a ROCm container."
+echo "Below is what it will do:"
+echo "  1. Pull the latest ROCm docker image;"
+echo "  2. Prepare a sample traing data set in a temporary directory;"
+echo "  3. Train $MODEL inside the ROCm container;"
+echo "  4. Clean up the temoprary directory created and exit."
+echo "Please press any key to start, or ESC to exit."
+
+# Read a key press
+read -n 1 -s KEY
+if [ "$KEY" == $'\e' ] ; then
+  echo "Exit. Did nothing."
+  exit 0
+fi
+
+# Get the folders
+SCRIPTPATH=$(dirname $(realpath $0))
+CODE_DIR=$SCRIPTPATH/..
+CODE_DIR_INSIDE=/data/code
+
+# Pull the docker image
+echo 
+echo "=== Docker pulling image $IMAGE and start container ==="
+docker pull $IMAGE
+CTNRNAME=RunDocker
```

Comment:
Rename container to something more clear. Maybe ROCMDockerContainer.

Reply:
It is just the name for the container, which only runs for a few minutes and it is gone. But feel free to change it to anything that makes a better sense.